content/en/database_monitoring/guide/sql_deadlock.md

```diff
@@ -42,6 +42,9 @@ Supported Agent versions
   ON SERVER
   ADD EVENT sqlserver.xml_deadlock_report
   ADD TARGET package0.ring_buffer
+  (
+  SET MAX_MEMORY = 1024
+  )
```

Comment:
Syntax looks a little weirdly formatted. Is there a reason for this or should the SET be indented?